portal/src/BottomActionContainer.module.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  @apply h-full;
+  @apply flex flex-col;
+  @apply overflow-auto;
+}
+
+.header {
+}
+
+.contentExpand {
+  @apply flex-1;
+}
+
+.footer {
+}
```

Comment:
The <div class="header"> and <div class="footer"> seems redundant. Or you document a reason to have them?

Reply:
Right, just removed